src/www/ui/core-auth.php

```diff
@@ -289,7 +289,8 @@ function checkUsernameAndPassword($userName, $password)
         $Email = $this->authExternal['emailAuthExternal'];
         /* Set default list of agents when a new user is created */
         $agentList = $GLOBALS['SysConf']['EXT_AUTH']['CONF_EXT_AUTH_NEW_USER_AGENT_LIST'];
-        add_user($User, $Desc, $Hash, $Perm, $Email, $Email_notify, $agentList, $Folder);
+        add_user($User, $Desc, $Hash, $Perm, $Email, $Email_notify,
+          $GLOBALS['SysConf']['EXT_AUTH']['CONF_EXT_AUTH_NEW_USER_UPLOAD_VISIBILITY'], $agentList, $Folder);
```

Comment:
I think it will be better if we can fetch this value from the DB rather than the `.conf` file which is easier to update.

The value can be fetched with `$GLOBALS['SysConf']['SYSCONFIG']['UploadVisibility']`.

![image](https://user-images.githubusercontent.com/18077542/123087779-8c32c680-d442-11eb-9421-c246e951e4f6.png)

Reply:
oh, of course, thanks. Didn't realize that was configurable in the web interface.

I guess the whole ext auth settings should also be available from the GUI, that would be a nice "good first issue"